controllers/compaction/reconciler.go

```diff
@@ -44,7 +44,8 @@ import (
 
 const (
 	// DefaultETCDQuota is the default etcd quota.
-	DefaultETCDQuota = 8 * 1024 * 1024 * 1024 // 8Gi
+	DefaultETCDQuota                 = 8 * 1024 * 1024 * 1024 // 8Gi
+	DefaultMetricsScrapeWaitDuration = "60s"
```

Comment:
@abdasgupta the default value for this should still be zero, since druid can still be used in non-gardener use-cases, ie, open-source usage.

The value for this should come as a flag to druid (compaction controller config) and should be set by gardenlet to `60s`, since gardenlet also configures prometheus to use a `60s` scrape interval.

Reply:
Done